styleguide/components/31-molecules/blocks/social-media-block/_social-media-block.scss

```diff
@@ -6,6 +6,10 @@
   min-height: 2rem;
   margin-bottom: 1rem;
 
+  @include phablet {
+    width: 100%;
```

Comment:
Shouldn't this be 100% below phablet too?

Reply:
Fixed.